docs/current/guides/snippets/use-secrets/host-env/main.py

```diff
@@ -0,0 +1,24 @@
+import sys
+
+import anyio
+import dagger
+
+async def main():
+    async with dagger.Connection(dagger.Config(log_output=sys.stderr)) as client:
+        # read secret from host variable
+        secret = client.host().env_variable("GH_SECRET").secret()
```

Comment:
I thought we wanted to deprecate `Host.EnvVariable.Secret`:
```suggestion
        secret = client.set_secret("GH_SECRET", os.environ.get("GH_SECRET", ""))
```
Maybe this hasn't been decided, need to find the reference for that. \cc @dolanor

Reply:
The problem was mostly `file.secret()` as it leaked the plaintext in the pipeline.
I think I could make it secure again, though. It will avoid breaking the API if we still want to keep those. Or maybe create another func/method to abstract that. (ref: https://github.com/dagger/dagger/issues/4403#issue-1538458042)

Reply:
Updated in guide